docs/animations/Fade.md

```diff
@@ -1,93 +1,87 @@
 # Fade
 
-The **Fade animation behavior** fades objects, in and out, over time.
+The Fade animation fades objects, in and out, over time. Fade animation is applied to all the XAML elements in its parent control/panel. Fade animation doesn't affect the functionality of the control.
 
 ## Syntax
 
-```xml
-
-    <behaviors:Fade x:Name="FadeBehavior>" 
-                Value="0.5" 
-                Duration="1500" 
-                Delay="500" 
-                AutomaticallyStart="True">
-    </behaviors:Fade>
+**In XAML**
 
+```xml
+<Page ...
+    xmlns:interactivity="using:Microsoft.Xaml.Interactivity"  
+    xmlns:behaviors="using:Microsoft.Toolkit.Uwp.UI.Animations.Behaviors"/>
+
+<interactivity:Interaction.Behaviors>
+    <behaviors:Fade x:Name="FadeBehavior" 
+            Value="0.5" 
+            Duration="2500" 
+            Delay="250" 
+            AutomaticallyStart="True"/>
+</interactivity:Interaction.Behaviors>
 ```
 
-or directly from code:
+**In Code Behind**
 
 ```csharp
-
-    MyRectangle.Fade((float)Value, Duration, Delay);
-
+MyUIElement.Fade(value: 0.5f, duration: 2500, delay: 250, easingType: EasingType.Default).Start();
+await MyUIElement.Fade(value: 0.5f, duration: 2500, delay: 250, easingType: EasingType.Default).StartAsync();  //Fade animation can be awaited
 ```
 
-## Properties
-
-| Property Name | Type | Description |
-| --- | --- | --- |
-| Value | float | The amount to fade an element. The value should be between 0.0 and l.0 |
-| Duration | double | The number of milliseconds the animation should run for |
-| Delay | double | The number of milliseconds before the animation is started |
-
-## Chaining animations
+## Sample Output
 
-Behavior animations can also be chained and awaited.
+![Fade Behavior animation](https://github.com/Vijay-Nirmal/UWPCommunityToolkit/blob/DocImprovements/docs/resources/images/Animations/Fade/Sample-Output.gif)
 
-```csharp
+## Properties
 
-    Element.Rotate(value: 30f, duration: 0.3).StartAsync();
 
-    await Element.Rotate(value: 30f, duration: 0.3).StartAsync();
 
-    var anim = element.Rotate(30f).Fade(0.5).Blur(5);
-    anim.SetDurationForAll(2);
-    anim.Completed += animation_completed;
-    anim.StartAsync();
+### EasingType
```

Comment:
@nmetulev I have done with doc improvements. Waiting for your opinion about the above comment.
> Yes. I also thought of a doc for AnimationSet. But in that, we can't include EasingType because some animation doesn't have EasingType like Saturation, Light